auth0/auth0-spa-js
# [SDK-1516] Web Workers

### Description

Add a Web Worker for using and storing refresh tokens for renewing access tokens.

### References

https://auth0team.atlassian.net/wiki/x/a52VJ

### Testing

- Run `npm start`
- Visit http://localhost:3000
- Disable 'Use local storage'
- Enable 'Use refresh tokens'
- Disable 'Use token cache when fetching new tokens'
- Login then click 'Get access token'
- Check that a new token added to the 'Access tokens' box


### Checklist

- [x] The correct base branch is being used, if not `master` (`release/rtr`)
